test/blackbox-tests/test-cases/toplevel-integration/run.t

```diff
@@ -0,0 +1,9 @@
+Test toplevel-init-file on empty project
+----------------------------------------------------
+  $ dune toplevel-init-file
+  Info: Creating file dune-project with this contents:
+  | (lang dune 2.1)
+  | (name test)
```

Comment:
Could you write a `dune-project` file explicitly here? If we let dune create one, then when we bump the latest version of the dune language this test will have to be updated. BTW, you can create files easily from the `run.t` file directly via:

```
  $ cat >dune-project <<EOF
  > (lang dune 2.1)
  > EOF
```

It's not mandatory, but personally I prefer this method as it makes the `run.t` file more self-contained.

Reply:
Ah, nice. Sure.